app/Http/Controllers/MeetupController.php

```diff
@@ -71,6 +73,6 @@ public function sendEmailOnMeetupCreation(Meetup $meetup)
         Mail::send('emails.meetup-created', compact('user', 'meetupDetails'), function ($message) use ($user, $recipients) {
```

Comment:
It would have been easier if you used `array_push`.

..but every environment variable works for staging and all PR builds.